### PR TITLE
Add View Command

### LINKED ISSUE #78: As a user, I want to view the record of a specific client
so that I can see the details of a client beyond what is shown in the list view

### LINKED ISSUE #84: List items are not properly highlighted
**Describe the bug**
List items are not properly highlighted after PR #80. The PersonCards are highlighted on hover, but does not stay selected after it is clicked or when user use arrow keys to navigate between items

### PR DESCRIPTION
**Pull request by mongj**
Closes #78 
Fixes #84

Implements a new view command which displays the selected client's details on the right pane.

New issues relating to implementation of the view command which are not resolved in this PR: #83, #85

### CODE REVIEW

File: src/main/java/careconnect/commons/core/GuiSettings.java
Comment (celeschai) [inline src/main/java/careconnect/commons/core/GuiSettings.java:1]: Good attention to details! Window size should be larger for users by default with additional panes.

Comment (celeschai) [inline src/main/java/careconnect/commons/core/GuiSettings.java:1]: Good attention to details! Window size should be larger for users by default with additional panes.

File: src/main/java/careconnect/ui/PersonDetailCard.java
Comment (celeschai) [inline src/main/java/careconnect/ui/PersonDetailCard.java:1]: The hyperlink function will be very helpful for users to find directions to their clients!

File: src/main/java/careconnect/ui/PersonDetailFallback.java
Comment (celeschai) [inline src/main/java/careconnect/ui/PersonDetailFallback.java:1]: I like your setting up of a fallback!

**Review by celeschai — APPROVED** [review]
Really good work!

**Comment by SuhailLoya** [comment]
Awesome Possom!

**Pull request merged**